src/mongo/db/storage/csv_file.cpp

```diff
@@ -303,34 +313,23 @@ void CsvFileInput::appendTo<CsvFieldType::kOid>(BSONObjBuilder& builder,
                                                 const std::string& fieldName,
                                                 const std::string_view& field) {
     constexpr size_t kLengthOidValue = 24;
-    constexpr size_t kOidTypeStr = 8;                      // Length of 'objectid'.
-    constexpr size_t kOidTypeStrPrefix = kOidTypeStr + 2;  // Length of 'objectid(\"'.
-
-    std::string mutableData{field.begin(), field.end()};
-    if (field[0] == 'O' || field[0] == 'o') {
-        std::transform(
-            mutableData.begin(), mutableData.begin() + kOidTypeStr, mutableData.begin(), ::tolower);
-
-        if (mutableData.substr(0, kOidTypeStrPrefix) != "objectid(\"") {
-            _ioStats->incInvalidOid();
-            builder.appendNull(fieldName);
-            return;
-        }
 
-        mutableData = mutableData.substr(kOidTypeStrPrefix, kLengthOidValue);
-    } else if (field[0] == '\"') {
-        mutableData = mutableData.substr(1, kLengthOidValue);
-    }
+    // checking if the oid is enclosed by double quote or not
+    std::string_view fieldData = field.front() == '\"' && field.back() == '\"'
+        ? std::string_view(field.begin() + 2, field.end() - 2)
```

Comment:
Why `+2` & `-2`? Could you give an example so that readers can understand this case more easily?